connector_redmine/README.rst

```diff
@@ -0,0 +1,67 @@
+Redmine Connector
+=================
+
+Base connector module for Redmine.
+It Allows to authenticate to a Redmine instance using the REST API.
```

Comment:
It allows the authentication to a Redmine instance using the REST API.
It also defines a method getUser that searches for the Redmine user related
to the Odoo user. 
Please be aware that user login must be the same in both systems.